Meta/run.py

```diff
@@ -0,0 +1,978 @@
+                "ne2k_isa,netdev=breh",
+                *config.kernel_and_initrd,
+                "-append",
+                kernel_cmdline,
+            ]
+        case MachineType.QEMU35Grub:
+            return [
+                config.qemu_binary or "",
+                *common_arguments,
+                *virtualization_arguments,
+                *config.network_default_device,
+            ]
+        case MachineType.Limine:
+            return [
+                config.qemu_binary or "",
+                *common_arguments,
+                *virtualization_arguments,
+            ]
+        case MachineType.CI:
+            if config.architecture == Arch.Aarch64:
+                return [
+                    config.qemu_binary or "",
+                    *config.extra_arguments,
+                    *virtualization_arguments,
+                    *config.boot_drive,
+                    "-M",
+                    "raspi3b",
+                    "-d",
+                    "guest_errors",
+                    "-no-reboot",
+                    "-nographic",
+                    "-monitor",
+                    "none",
+                    "-display",
+                    "none",
+                    "-serial",
+                    "file:debug.log",
+                    "-serial",
+                    "stdio",
+                    *config.kernel_and_initrd,
+                    "-append",
+                    kernel_cmdline,
+                ]
+            else:
+                return [
+                    config.qemu_binary or "",
+                    *config.extra_arguments,
+                    *virtualization_arguments,
+                    *config.boot_drive,
+                    "-m",
+                    config.ram_size,
+                    "-cpu",
+                    config.qemu_cpu,
+                    "-d",
+                    "guest_errors",
+                    "-no-reboot",
+                    "-nographic",
+                    "-display",
+                    "none",
+                    "-device",
+                    "ich9-ahci",
+                    "-debugcon",
+                    "file:debug.log",
+                    *config.kernel_and_initrd,
+                    "-append",
+                    kernel_cmdline,
+                ]
+        case MachineType.Default:
+            return [
+                config.qemu_binary or "",
+                *common_arguments,
+                *virtualization_arguments,
+                *packet_logging_arguments,
+                *config.network_default_device,
+                *config.extra_arguments,
+                *config.kernel_and_initrd,
+                "-append",
+                kernel_cmdline,
+            ]
+
+
+def configure_and_run():
+    config = Configuration()
+    config.kvm_usable = kvm_usable()
+    config.qemu_kind = determine_qemu_kind()
+    config.architecture = determine_serenity_arch()
+    config.machine_type = determine_machine_type()
+
```

Comment:
These three don't take the config as argument while the others do. would be nice to be consistent.

Reply:
I think all independent configuration determining functions that just update one variable should behave like these, will change it.